db/crud.go

```diff
@@ -1621,6 +1622,11 @@ func (db *DatabaseContext) assignSequence(ctx context.Context, docSequence uint6
 			if err != nil {
 				return unusedSequences, err
 			}
+			numSkippedSequences := docSequence - doc.Sequence
```

Comment:
What we want to evaluate here is the difference between the sequence we attempted to assign on line 1611 (which was where the sequence allocator was at that point in time) and the sequence we get back from nextSequenceGreaterThan on line 1621.  
The warning should include all three sequences (line 1611, doc.Sequence, and line 1621)

Reply:
I think I addressed this, but I'm a bit confused and the warning messages could probably read better. I'd appreciate feedback here.